### PR TITLE
coqdoc: update refman and stdlib URLs

### PR DESCRIPTION
**Pull request by brandenburg**
The reference manual and the standard library documentation have moved on the Coq homepage. The URLs are now `https://coq.inria.fr/doc/V${VERSION}/refman/`  and `https://coq.inria.fr/doc/V${VERSION}/stdlib/`, respectively. 

While there is an automatic redirect from the old URL to the new one, the redirect breaks the anchors that `coqdoc` uses to refer to specific symbols. To avoid the problem, this patch updates the configuration to link to the actual URL instead to avoid the anchor-breaking redirect. 

See also: https://github.com/coq/coq.github.io/issues/235

### CODE REVIEW

**Comment by brandenburg** [comment]
This is a trivial change. I'm not sure who needs to review this, but hopefully it should not take more than a quick glance. Could someone with write access please take a look?

**Review by ppedrot — APPROVED** [review]
This looks indeed OK.

**Comment by ppedrot** [comment]
Although not officially maintainer of the configure file, I'll merge this soon if nobody complains.

**Comment by SkySkimmer** [comment]
Should this go in 8.19?

**Comment by ppedrot** [comment]
In any case I'm putting this in the 8.19 milestone, @SkySkimmer take your decision and move it forward if you judge it so.

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**
